terraform-modules/cid-dashboards/README.md

```diff
@@ -0,0 +1,151 @@
+# CID Terraform Module: cid-dashboards
+
+Terraform module to deploy CID dashboards. This module is a wrapper around CloudFormation
+to allow you to deploy CID dashboards using your existing Terraform workflows. Under the
+hood, the module will deploy a CloudFormation stack which will provision the necessary
+resources and a custom Lambda function to create the dashboards using `cid-cmd`.
+
+## Before You Start
+
+  - Existing S3 bucket to upload the CloudFormation template
```

Comment:
Not clear what key to use? Can we provide this in a readme?

Reply:
Can you clarify which key you are referring to?